jekyll/_docs/yarn.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: classic-docs
+title: "Using Yarn (the npm replacement) on CircleCI"
```

Comment:
Why define Yarn here? I'm assuming people are already searching for Yarn and know what it is. Change to: "Using Yarn on CircleCI"